packages/peregrine/lib/store/reducers/cart.js

```diff
@@ -31,10 +35,9 @@ const reducerMap = {
             cartId: String(payload)
         };
     },
-    [actions.getDetails.request]: (state, { payload }) => {
+    [actions.getDetails.request]: state => {
         return {
             ...state,
-            cartId: String(payload),
```

Comment:
The only thing setting `cartId` should be `getCart` action.

Reply:
For context, why did this action accept `cartId` as a payload in the first place, and why did the reducer take this opportunity to set it in state?

Reply:
No idea. Requests are only meant to set `loading `states in redux right now. Even that will eventually move to UI I believe. At that point all we will are about are `.receive` sync actions.